yapapi/golem.py

```diff
@@ -85,6 +136,55 @@ async def run_service(
             payload specified by the `get_payload()` method of `service_class` is used
         :param expiration: optional expiration datetime for the service
         :return: a `Cluster` of service instances
+
+        example usage:
```

Comment:
This one is pretty lengthy for a docstring, but I suppose it's better to include an example. :shrug:

Reply:
:D well, it's your example so it is already included as an _example_ ...